exchanges/btcmarkets/btcmarkets_websocket.go

```diff
@@ -365,3 +380,41 @@ func (b *BTCMarkets) Subscribe(channelsToSubscribe []stream.ChannelSubscription)
 	b.Websocket.AddSuccessfulSubscriptions(channelsToSubscribe...)
 	return nil
 }
+
+// checksum provides assurance on current in memory liquidity
+func checksum(ob *orderbook.Base, checksum uint32) error {
+	check := crc32.ChecksumIEEE([]byte(concat(ob.Bids) + concat(ob.Asks)))
+	if check != checksum {
+		return fmt.Errorf("%s %s %s ID: %v expected: %v but received: %v %w",
+			ob.Exchange,
+			ob.Pair,
+			ob.Asset,
+			ob.LastUpdateID,
+			checksum,
+			check,
+			errChecksumFailure)
+	}
+	return nil
+}
+
+// concat concatenates price and amounts together for checksum processing
+func concat(liquidity orderbook.Items) string {
+	length := 10
+	if len(liquidity) < 10 {
+		length = len(liquidity)
+	}
+	var concat string
```

Comment:
Super minor, but could you change the var name so it is not the same as the function name? Or vice versa